src/azimuth/control/util.c

```diff
@@ -19,24 +19,68 @@
 
 #include "azimuth/control/util.h"
 
+#include <SDL_filesystem.h>
+
 #include <assert.h>
 #include <stdbool.h>
 #include <stdlib.h>
 
+#if !defined(__WINDOWS__) && !defined(__APPLE__)
+#include <string.h>
+#include <sys/stat.h>
+#include <wordexp.h>
+#endif
+
 #include "azimuth/gui/audio.h"
 #include "azimuth/state/save.h"
 #include "azimuth/system/resource.h"
 #include "azimuth/util/prefs.h"
 #include "azimuth/util/string.h"
 #include "azimuth/view/prefs.h"
 
+#define ORG_NAME NULL
```

Comment:
Short version: Can we replace `NULL` with `""` here?

Long version: I tested out this change on my Mac and got a segfault:

```
==83589==ERROR: AddressSanitizer: SEGV on unknown address 0x000000000000 (pc 0x7fff7e18d6f2 bp 0x7ffeef5daf90 sp 0x7ffeef5daf90 T0)
==83589==The signal is caused by a READ memory access.
==83589==Hint: address points to the zero page.
    #0 0x7fff7e18d6f1 in _platform_strlen (libsystem_platform.dylib:x86_64+0x16f1)
    #1 0x1009ebed8 in wrap_strlen (libclang_rt.asan_osx_dynamic.dylib:x86_64h+0x17ed8)
    #2 0x10097de95  (SDL2:x86_64+0xbbe95)
    #3 0x100737566 in az_load_preferences util.c:80
    #4 0x10073a0f0 in main main.c:70
    #5 0x7fff7dfa53d4 in start (libdyld.dylib:x86_64+0x163d4)
```

I checked the latest source code for `SDL_GetPrefPath` and looks like it should handle `NULL` for the `org` param just fine, by treating it the same as empty string.  However, I happen to have an older version of SDL2 installed (v2.0.5, from 2016-ish), and as far as I can tell, `SDL_GetPrefPath` wasn't changed to handle `NULL` until [this commit](https://github.com/libsdl-org/SDL/commit/658975f3817d6fee208e260f12549acdb4696fc9) in 2017.

Since the [docs for `SDL_GetPrefPath`](https://wiki.libsdl.org/SDL_GetPrefPath) don't seem to explicitly guarantee that `NULL` will work for the `org` param, and newer SDL2 versions seem to treat it the same as empty string anyway, I suggest we just `#define ORG_NAME ""` here.  I tested that on my Mac (with SDL2.0.5) and it seems to work fine (and still seems to give the same directory as the old code that this PR is replacing).  I haven't tested on Linux or Windows, though.

Reply:
Tested it and `""` works the same as `NULL` on Linux & Windows.